infra/modules/azure_monorepo_single_env_starter_pack/github_branch_rules.tf

```diff
@@ -0,0 +1,25 @@
+resource "github_branch_default" "main" {
+  repository = github_repository.this.name
+  branch     = "main"
+}
+
+resource "github_branch_protection" "main" {
+  repository_id = github_repository.this.name
+  pattern       = "main"
+
```

Comment:
```suggestion
  enforce_admins         = true
  allows_deletions       = false
  allows_force_pushes    = false
  require_signed_commits = true
```

Reply:
What is `enforce_admins` about?

Reply:
enforces status checks for repository administrators.